src/models/slices/filters.ts

```diff
@@ -1,8 +1,18 @@
-import { DisplayableFacet, SortBy } from '@yext/answers-core';
+import { DisplayableFacet } from '@yext/answers-core';
 import { SelectableFilter } from '../utils/selectablefilter';
 
+/**
+ * Maintains the current state of facets and filters in the application.
+ */
 export interface FiltersState {
+  /**
+   * The collection of possible static filters that can be applied to the
+   * search results and whether they are currently selected.
```

Comment:
Minor nit: Here and below, the phrase "whether they are currently selected" is a bit misleading. It makes it sound as if the whole array is selected or unselected. It may be more accurate to say "and which are currently selected".

Reply:
Sure that sounds good

Reply:
Updated